state/indexer/doc.go

```diff
@@ -0,0 +1,74 @@
+/*
+Package indexer defines Tendermint's block and transaction event indexing logic.
+
+Currently, Tendermint supports two primary means of block and transaction event
+indexing:
+
+1. A key-value sink via an embedded database with a proprietary query language.
+2. A Postgres-based sink.
+
+Recall, an ABCI application can emit events during block and transaction execution
+in the form of <abci.Event.Type>.<abci.EventAttributeKey>=<abci.EventAttributeValue>,
+e.g. 'transfer.amount=10000'.
+
+An operator can enable one or both of the supported indexing sinks via the
+'tx-index.indexer' Tendermint configuration.
+
+Example:
+
+	[tx-index]
+	indexer = ["kv", "psql"]
+
+If an operator wants to completely disable indexing, they may simply just provide
+the "null" sink option in the configuration. All other sinks will be ignored if
+"null" is provided.
+
+If indexing is enabled, the indexer.Service will iterate over all enabled sinks
```

Comment:
```suggestion
If indexing is enabled, the indexer service will iterate over all enabled sinks
```

or perhaps "indexer.Service", but I don't know how necessary it is to spell out the type vs. just describing it. YMMV.

Reply:
No need for this change.